ee/server/service/mdm.go

```diff
@@ -0,0 +1,89 @@
+package service
+
+import (
+	"context"
+	"encoding/json"
+	"io"
+	"io/ioutil"
+
+	"github.com/fleetdm/fleet/v4/pkg/fleethttp"
+	"github.com/fleetdm/fleet/v4/server/contexts/ctxerr"
+	"github.com/fleetdm/fleet/v4/server/fleet"
+	apple_mdm "github.com/fleetdm/fleet/v4/server/mdm/apple"
+	"github.com/micromdm/nanodep/client"
+	"github.com/micromdm/nanodep/storage"
+)
+
+func (svc *Service) GetAppleBM(ctx context.Context) (*fleet.AppleBM, error) {
+	if err := svc.authz.Authorize(ctx, &fleet.AppleBM{}, fleet.ActionRead); err != nil {
+		return nil, err
+	}
+
+	// if there is no apple bm config, fail with a 404
+	if !svc.config.MDM.IsAppleBMSet() {
+		return nil, notFoundError{}
+	}
+
+	appCfg, err := svc.AppConfig(ctx)
+	if err != nil {
+		return nil, err
+	}
+	tok, err := svc.config.MDM.AppleBM()
+	if err != nil {
+		return nil, err
+	}
+
+	appleBM, err := getAppleBMAccountDetail(ctx, svc.depStorage)
+	if err != nil {
+		return nil, err
+	}
+
+	// fill the rest of the AppleBM fields
+	appleBM.RenewDate = tok.AccessTokenExpiry
+	// TODO: default team will have to be set when https://github.com/fleetdm/fleet/issues/8733
+	// is implemented.
+	appleBM.DefaultTeam = ""
+	// TODO: is that path from the prototype `/mdm/apple/mdm` still the path we
+	// want to use for the official feature? AIUI, this is the path hit for
+	// requests to enroll devices (or at least that's what I think "check-in"
+	// is?) and execute MDM commands.
+	appleBM.MDMServerURL = appCfg.ServerSettings.ServerURL + apple_mdm.MDMPath
```

Comment:
Anyone can confirm this ^ ?

Reply:
Issue: https://github.com/fleetdm/fleet/issues/8999